virttest/cpu.py

```diff
@@ -50,15 +50,15 @@
                               "Haswell", "Haswell-noTSX", "IvyBridge",
                               "SandyBridge", "Westmere", "Nehalem",
                               "Penryn", "Conroe"]}
-CPU_TYPES_RE = {"EPYC-Rome": "rdpid,wbnoinvd,stibp,clwb,umip",
+CPU_TYPES_RE = {"EPYC-Milan": "ibrs,pcid,ssbd,erms,fsrm,invpcid,pku",
```

Comment:
From libvirt side, EPYC-Milan have 2 another flags `svme-addr-chk` and `amd-ssbd` when compare with EPYC-Rome,  do we need them ?

Reply:
> From libvirt side, EPYC-Milan have 2 another flags `svme-addr-chk` and `amd-ssbd` when compare with EPYC-Rome, do we need them ?

Yes, but svme-addr-chk can't find on host "Flags", so we won't use it to check.

Reply:
Thanks for your detailed info offline, got it now.